src/formatPkg.ts

```diff
@@ -157,7 +159,8 @@ export default function formatPkg(pkg: GetPackage): RawPkg | undefined {
     gitHead: githubRepo ? githubRepo.head : null, // remove this when we update to the new schema frontend
     readme: pkg.readme,
     owner,
-    deprecated: cleaned.deprecated !== undefined ? cleaned.deprecated : false,
+    deprecated: isDeprecated,
+    deprecatedReason: isDeprecated ? String(cleaned.deprecated) : null,
```

Comment:
I'm not sure we can afford breaking changes on the index, can you keep `deprecated` as is, and add a new field `isDeprecated` for the boolean form?

Reply:
We could, we are providing this for free and it was not good. 
However we can indeed provide a grace period for the transition, by adding not changing `deprecated` for the moment.
Do we have a list of authorised user somewhere to contact?

Reply:
I have added the new field and put back deprecated